{"uid":"6e5473cc3581175a","name":"Verify that all transition operations have the expected data","fullName":"api.2-disruptive.test_transitions_on.tavern#Verify that all transition operations have the expected data","historyId":"954d1a656b2d56809aff55bf542986d7","time":{"start":1678226979675,"stop":1678226979811,"duration":136},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"passed","steps":[{"name":"Stage 0: Retrieve all transition operation data","time":{"start":1678226979681,"stop":1678226979811,"duration":130},"status":"passed","steps":[],"attachments":[{"uid":"21b8d99f7ff3791a","name":"stage_yaml","source":"21b8d99f7ff3791a.yaml","type":"application/yaml","size":2367},{"uid":"46c4ccd511ff8cd8","name":"rest_request","source":"46c4ccd511ff8cd8.yaml","type":"application/yaml","size":124},{"uid":"5b4598504b0980b7","name":"rest_response","source":"5b4598504b0980b7.yaml","type":"application/yaml","size":19670}],"parameters":[],"stepsCount":0,"attachmentsCount":3,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":3,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"cray-power-control-hmth-test"},{"name":"suite","value":"disruptive"},{"name":"host","value":"22ad509d3223"},{"name":"thread","value":"9-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"api.2-disruptive.test_transitions_on.tavern"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":13,"unknown":0,"total":14},"items":[{"uid":"7a8687b72cbed04","status":"passed","time":{"start":1678224997335,"stop":1678224997476,"duration":141}},{"uid":"ee63b7ea28021f79","status":"passed","time":{"start":1678221630769,"stop":1678221630842,"duration":73}},{"uid":"89680e165df57e72","status":"passed","time":{"start":1678217940581,"stop":1678217940660,"duration":79}},{"uid":"13e31e6be16423c","status":"passed","time":{"start":1676400739254,"stop":1676400739469,"duration":215}},{"uid":"707e3870c5ca5742","status":"passed","time":{"start":1676325524194,"stop":1676325524331,"duration":137}},{"uid":"cb9b0c83a860a74","status":"passed","time":{"start":1674515480521,"stop":1674515480652,"duration":131}},{"uid":"ed9a3b490927c74d","status":"passed","time":{"start":1674505627625,"stop":1674505627759,"duration":134}},{"uid":"75f9bce45d23a181","status":"passed","time":{"start":1674500225339,"stop":1674500225471,"duration":132}},{"uid":"ffa5d8fb741fd1b2","status":"passed","time":{"start":1674257721217,"stop":1674257721354,"duration":137}},{"uid":"acf740812abb29a1","status":"passed","time":{"start":1674255186379,"stop":1674255186556,"duration":177}},{"uid":"67df72cc04b9af61","status":"failed","statusDetails":"tavern.util.exceptions.TestFailError: Test 'Retrieve all transition operation data' failed:\n- Error calling validate function '<function validate_pykwalify at 0x7f0b0c336c10>':\n    Traceback (most recent call last):\n      File \"/usr/lib/python3.9/site-packages/tavern/schemas/files.py\", line 106, in verify_generic\n        verifier.validate()\n      File \"/usr/lib/python3.9/site-packages/pykwalify/core.py\", line 194, in validate\n        raise SchemaError(u\"Schema validation failed:\\n - {error_msg}.\".format(\n    pykwalify.errors.SchemaError: <SchemaError: error code 2: Schema validation failed:\n     - Enum 'ForceOff' does not exist. Path: '/transitions/0/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/1/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/3/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/4/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/5/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/6/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/9/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/11/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/18/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/19/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/20/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].\n     - Enum 'ForceOff' does not exist. Path: '/transitions/21/operation' Enum: ['Init', 'On', 'Off', 'Soft-Off', 'Force-Off', 'Soft-Restart', 'Hard-Restart'].: Path: '/'>\n\n    The above exception was the direct cause of the following exception:\n\n    Traceback (most recent call last):\n      File \"/usr/lib/python3.9/site-packages/tavern/response/base.py\", line 152, in _maybe_run_validate_functions\n        vf(response)\n      File \"/usr/lib/python3.9/site-packages/tavern/util/extfunctions.py\", line 98, in inner\n        result = func(response, *args, **kwargs)\n      File \"/usr/lib/python3.9/site-packages/tavern/testutils/helpers.py\", line 103, in validate_pykwalify\n        verify_generic(to_verify, schema)\n      File \"/usr/lib/python3.9/site-packages/tavern/schemas/files.py\", line 109, in verify_generic\n        raise BadSchemaError() from e\n    tavern.util.exceptions.BadSchemaError","time":{"start":1674080261426,"stop":1674080261491,"duration":65}},{"uid":"829b1ea24cc5b56d","status":"passed","time":{"start":1674074977922,"stop":1674074977942,"duration":20}},{"uid":"454c12bba4d656d5","status":"passed","time":{"start":1674074600371,"stop":1674074600387,"duration":16}}]},"tags":[]},"source":"6e5473cc3581175a.json","parameterValues":[]}